src/main/java/com/carleton/CapstoneSystem/restfulControllers/ProjectResource.java

```diff
@@ -72,4 +72,10 @@ public Response addStudents(@RequestBody ProjectDTO projectDTO) {
 
         return projectController.addStudents(projectDTO);
     }
+
+    @PostMapping("/addDeliverable")
```

Comment:
I think we need a new resource called SubmissionResource for this endpoint. The endpoint should take a submissionDTO. The submissionDTO has name, dueDate and projectDTO.

Reply:
This PR is for the front end, I will leave that up to you or Ali to decide.

Reply:
sounds good man 😄